Cargo.toml

```diff
@@ -9,3 +9,6 @@ members = [
   "futures-util",
   "futures-test",
 ]
+
+[patch.crates-io]
+pin-utils = { git = "https://github.com/Kroisse/pin-utils.git", branch = "pin" }
```

Comment:
Do we plan on publishing the needed changes here?

Reply:
These are actually already published-- I just have to update the path here.